modules/varnish/templates/default.vcl

```diff
@@ -275,6 +275,10 @@ sub mw_stash_cookie {
 }
 
 sub mw_evaluate_cookie {
+        if (req.url ~ "^/w/load\.php") {
+           unset req.http.Cookie; # load.php URLs should be specfic and sessions disabled, we can ignore cookies

```

Comment:
can this not use mw_stash_cookie like below? Also is this not triggered by the below?

Reply:
That's what I thought also but was told above that it wasn't.